Tests/js/views/fields/assets/ez-relation-editview-tests.js

```diff
@@ -0,0 +1,419 @@
+                    that.jsonVersion, variables.version,
+                    "The version should be available in the field edit view template"
+                );
+                Y.Assert.areSame(
+                    that.jsonContentType, variables.contentType,
+                    "The contentType should be available in the field edit view template"
+                );
+                Y.Assert.areSame(
+                    fieldDefinition, variables.fieldDefinition,
+                    "The fieldDefinition should be available in the field edit view template"
+                );
+                Y.Assert.areSame(
+                    that.field, variables.field,
+                    "The field should be available in the field edit view template"
+                );
+
+                Y.Assert.areSame(expectRequired, variables.isRequired);
+                return '';
+            };
+            this.view.render();
+        },
+
+        "Test not required field": function () {
+            this._testAvailableVariables(false, false);
+        },
+
+        "Test required field": function () {
+            this._testAvailableVariables(true, true);
+        },
+
+        "Test validate no constraints": function () {
+            var fieldDefinition = this._getFieldDefinition(false);
+
+            this.view.set('fieldDefinition', fieldDefinition);
+            this.view.render();
+            this.field = {fieldValue: {destinationContentId: 45}};
+
+            this.view.validate();
+            Y.Assert.isTrue(
+                this.view.isValid(),
+                "An empty relation is valid"
+            );
+            Y.Assert.areSame(
+                this.view.get('field').fieldValue.destinationContentId,
+                this.view.get('destinationContentId'),
+                "The destinationContentId of the field value should be the same than the attribute"
+            );
+        },
+
+        "Test validate required": function () {
+            var fieldDefinition = this._getFieldDefinition(true);
+
+            this.view.set('fieldDefinition', fieldDefinition);
+            this.view._set('destinationContentId', null);
+            this.view.validate();
+            this.view.render();
+
+            Y.Assert.isFalse(
+                this.view.isValid(),
+                "An empty relation is NOT valid"
+            );
+        },
+
+        "Should fire the loadFieldRelatedContent event": function () {
+            var loadContentEvent = false,
+                that = this;
+
+            this.view.on('loadFieldRelatedContent', function (e) {
+                Y.Assert.areSame(
+                    that.fieldDefinitionIdentifier,
+                    e.fieldDefinitionIdentifier,
+                    "fieldDefinitionIdentifier is the same than the one in the field"
+                );
+                loadContentEvent = true;
+            });
+            this.view._set('destinationContentId', 45);
```

Comment:
given how the view is initialized (with a non empty field), this line does not seem necessary.